ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/network/Util.kt

```diff
@@ -0,0 +1,63 @@
+package ai.nnstreamer.ml.inference.offloading.network
+
+import ai.nnstreamer.ml.inference.offloading.App
+import android.app.Service
+import android.net.ConnectivityManager
+import android.util.Log
+import java.net.Inet4Address
+import java.net.ServerSocket
+import kotlin.concurrent.thread
+
+const val TAG = "NetworkUtil"
+
+/**
+ * Returns the IP address based on whether the app is running on an emulator or not.
+ *
+ * @param isRunningOnEmulator Indicates whether the app is running on an emulator.
+ * @return The IP address as a string.
+ * @todo Add an ApplicationContext Parameter.
```

Comment:
Why?

Reply:
I copied the part from the original comment, is it okay if I delete it?

Reply:
Sure!